controllers/shooter_controller.py

```diff
@@ -0,0 +1,61 @@
+# from magicbot import StateMachine, state
+from components.shooter import Shooter
+import rev
+
+# class ShooterController(StateMachine):
+class ShooterController:
+    shooter: Shooter
+
+    def __init__(self) -> None:
+        self.outer_rpm_constant = 5000
+        self.centre_rpm_constant = 2200
+        self.outer_rpm = 5000
+        self.centre_rpm = 2200
+    
+    def on_enable(self) -> None:
+        self.stay_constant = False
+
+    def execute(self) -> None:
+        self.centre_pid.setReference(self.centre_rpm, rev.ControlType.kVelocity)
+        self.outer_pid.setReference(self.outer_rpm, rev.ControlType.kVelocity)
+
+    def setup(self) -> None:
+        self.configurePIDs()
+        self.shooter.loading_piston.setPulseDuration(1)
+
+    def configurePIDs(self) -> None:
+        self.centre_pid = self.shooter.centre_motor.getPIDController()
```

Comment:
Why is this in the high-level controller? This should not be accessing implementation details of our components.